private/model/api/shape.go

```diff
@@ -343,6 +355,11 @@ func (ref *ShapeRef) GoTags(toplevel bool, isRequired bool) string {
 	if ref.Deprecated || ref.Shape.Deprecated {
 		tags = append(tags, ShapeTag{"deprecated", "true"})
 	}
+
+	if ref.JSONValue {
+		tags = append(tags, ShapeTag{"jsonvalue", "true"})
```

Comment:
is this still needed if the shape(Ref).Type is a `jsonvalue`? I wouldn't expect it to be.